enclave-runtime/src/top_pool_execution.rs

```diff
@@ -178,3 +190,51 @@ where
 
 	Ok(())
 }
+
+/// Executes aura for the given `slot`
+fn exec_aura_on_slot<Authority, PB, SB, OCallApi, LightValidator, PEnvironment, ExtrinsicsFactory>(
+	slot: SlotInfo<PB>,
+	authority: Authority,
+	validator: &mut LightValidator,
+	extrinsics_factory: &ExtrinsicsFactory,
+	ocall_api: OCallApi,
+	proposer_environment: PEnvironment,
+	shards: Vec<ShardIdentifierFor<SB>>,
+) -> Result<()>
+where
+	// setting the public type is necessary due to some non-generic downstream code.
```

Comment:
I know this is copy pasted but maybe you know what this comment actually means.. i.e. if I understand it correctly this comment does not mean `public type` but rather `generic block hash` ?

Reply:
the comment is out of place - it's about the 'Public' associated type of `SignedBlock` (two lines further down). I can move the comment.